config/locales/time/es.yml

```diff
@@ -1,5 +1,22 @@
 ---
 es:
+  date:
+    month_names:
+    - 
+    - enero
+    - febrero
+    - marzo
+    - abril
+    - mayo
+    - junio
+    - julio
+    - agosto
+    - septiembre
+    - octubre
+    - noviembre
+    - diciembre
```

Comment:
Should these be capitalized? (Same question for French)

Reply:
I don't believe the months are capitalized in French and Spanish.

Reply:
since the Spanish translation has the month as the first word I should capitalize that.  Do you confirm on leaving French not capitalized?

Reply:
You are right about months in French not being capitalized. I confirm that we should leave them lowercase.

Reply:
Looks like in Spanish, the day comes first as well, so it should stay not capitalized, and we need to add some `de`s in there, such as `31 de diciembre de 2017`. Source: https://www.spanishdict.com/guide/dates-in-spanish